app/controllers/account/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Account::CommentsController < Account::AccountController
+  before_action :load_commentable
+
+  def create
+    @comments = @commentable.comments.order_desc
+    @comment = @comments.build(comments_params)
+    @result = @comment.save
```

Comment:
fyi: 
```
@comment.save
# works only on new records
if @comment.persisted?
  # record was saved
else
  # record wasn't saved
end
```
or
```
@comment.save
if @comment.new_record?
  # record wasn't saved
else
  # record was saved
end
```

Reply:
yes, you can do as Roman suggested to improve what I told before

@comment.save

and then instead of "if result" use "if @comment.persisted?". minor change needed